### PR TITLE
fix upload readme

### PR DESCRIPTION
**Pull request by sapirshuker**
## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/commands/upload/README.md [hunk 1]
```diff
@@ -95,6 +95,14 @@ Uploading classifiers to Cortex XSOAR is available from version 6.0.0 and up.
 
     uploads the pack to a XSIAM server. Must be used together with -z
 
+* **--console-log-threshold**
+
+    Minimum logging threshold for the console logger. Possible values: DEBUG, INFO, WARNING, ERROR.
+
+* **--file-log-threshold**
+
+    Minimum logging threshold for the file logger. Possible values: DEBUG, INFO, WARNING, ERROR.
```
Comment (GuyAfik) [inline demisto_sdk/commands/upload/README.md:104]: @tschanfeld can you please take a look?

**Review by GuyAfik — COMMENTED** [review]
don't need changelog, only docs changes

File: demisto_sdk/commands/upload/README.md (hunk 1 above)
Comment (tschanfeld) [reply demisto_sdk/commands/upload/README.md:104]: @GuyAfik this looks fine

**Review by GuyAfik — APPROVED** [review]
💯

**Review by tschanfeld — APPROVED** [review]

**Pull request merged**